test/org/loklak/http/ClientConnectionTest.java

```diff
@@ -12,7 +12,7 @@
     public void metaUrlExtractorTest() throws IOException {
         HashMap<String, String> linkMap = new HashMap<>();
 
-        linkMap.put("http://yacy.net", "http://yacy.net/en/index.html");
+        linkMap.put("http://tinyurl.com/loklak-server", "https://github.com/loklak/loklak_server/wiki");
```

Comment:
The test is for checking redirects from meta tag present on a webpage and not for `301` response codes. We need to check this with something similar.